### PR TITLE
Updated the `:guide intro` guide

### PR DESCRIPTION
**Pull request by renetapopova**
This PR updates the `:play intro` guide. When it is approved, I'll replicate the changes to the sidebar intro guide.

### CODE REVIEW

**Comment by renetapopova** [comment]
@OskarDamkjaer, the tests have failed as I haven't removed the tests related to the `:play intro` guide. I guess when you merge the PR that will make the `:play [built in guide]` commands automagically become `:guide` commands, you will also resolve that issue, and I will then rebase.

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 1]
```diff
@@ -22,31 +22,34 @@ import React from 'react'
...
+      <li>Exportable tabular results of any query result.</li>
       <li>
-        Developer focused, for writing and running graph queries with Cypher
-      </li>
-      <li>Exportable tabular results of any query result</li>
-      <li>
-        Graph visualization of query results containing nodes and relationships
+        Graph visualization of query results containing nodes and relationships.
       </li>
+      <li>Convenient exploration of the Neo4j's REST API.</li>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:41]: Convenient exploration of the Neo4j's HTTP API (REST).

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 2]
```diff
@@ -22,31 +22,34 @@ import React from 'react'
...
+    <p className="lead">
+      <em>Edit and execute Cypher statements and Browser commands</em>
+    </p>
     <p>
-      Editor pane is used to edit and run for Cypher statements and Browser
-      commands. Browser commands begin with <code>:</code>, for example{' '}
-      <code>:help</code>
+      Editor is the primary interface for entering and running Cypher statements
+      and Browser commands. Browser commands begin with <code>:</code>, for
+      example, <code>:help</code>.
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:52]: The Editor is the primary interface for entering and running Cypher statements
      and Browser commands. Browser commands begin with <code>:</code>, for
      example, <code>:help</code>.

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 3]
```diff
@@ -22,31 +22,34 @@ import React from 'react'
...
       </li>
+      <li>Convenient exploration of the Neo4j's REST API.</li>
     </ul>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s2">
-    <h3>Editor</h3>
-    <p className="lead">Command editing and execution</p>
+    <h3>Browser Editor (a.k.a. Editor)</h3>
+    <p className="lead">
+      <em>Edit and execute Cypher statements and Browser commands</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:47]: Edit and execute Cypher statements and Browser commands.

Dot or no dot ??

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 4]
```diff
@@ -79,60 +82,71 @@ const slides = [
...
     <video autoPlay loop muted playsInline>
-      <source src="./assets/images/Keystores.mp4" type="video/mp4" />
+      <source src="./assets/images/Keystrokes.mp4" type="video/mp4" />
     </video>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s3">
     <h3>Result frame</h3>
-    <p className="lead">Most recently executed command or Cypher query.</p>
+    <p className="lead">
+      <em>Most recently executed command or Cypher query</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:96]: Most recently executed command or Cypher query.

Dot or no dot ??

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 5]
```diff
@@ -79,60 +82,71 @@ const slides = [
...
     <p>
-      A result frame is created for each command execution, added to the top of
-      the stream to create a scrollable collection in reverse chronological
-      order.
+      A result frame is created for each execution and added to the top of the
+      stream to create a scrollable collection in reverse chronological order.
     </p>
     <ul>
-      <li>If a frame is pinned it will always stay in the same position. </li>
+      <li>A pinned frame always stays in the same position. </li>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:103]: `<li>A pinned frame always stays in the same position.</li>`

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 6]
```diff
@@ -79,60 +82,71 @@ const slides = [
...
       </li>
     </ul>
     <br />
     <img src="./assets/images/Stream.png" className="img-responsive padding5" />
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s4">
     <h3>Reusable frame</h3>
-    <p className="lead">Instead of the stream</p>
+    <p className="lead">
+      <em>Ammend your query in place</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:123]: Dot or no dot ?

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 7]
```diff
@@ -79,60 +82,71 @@ const slides = [
...
     <video autoPlay loop muted playsInline>
       <source src="./assets/images/ReusableFrame.mp4" type="video/mp4" />
     </video>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s5">
-    <h3>Sidebar: Database information</h3>
-    When Neo4j is installed, it is initiated with two databases - a{' '}
+    <h3>Sidebar: Database</h3>
+    <p className="lead">
+      <em>Database infromation</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:142]: Dot or no dot ?

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 8]
```diff
@@ -141,12 +155,15 @@ const slides = [
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s6">
     <h3>Sidebar: Favorites</h3>
+    <p className="lead">
+      <em>Quick way to save your queries</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:159]: Dot or no dot ?

File: src/browser/documentation/sidebar-guides/intro.tsx [hunk 9]
```diff
@@ -157,10 +174,18 @@ const slides = [
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s7">
     <h3>Sidebar: Project files</h3>
-    <p className="lead">Save cypher files to share with your colleagues</p>
-    Queries and commands can also be saved as <b>Project files</b>. Project
-    folder can be reached through Neo4j Desktop and your hard disk. Note this
-    feature is only available when Neo4j Browser is run in Desktop.
+    <p className="lead">
+      <em>Save cypher files to share with your colleagues</em>
```
Comment (martin-neotech) [inline src/browser/documentation/sidebar-guides/intro.tsx:178]: Dot or no dot?

**Review by martin-neotech — CHANGES_REQUESTED** [review]
Thanks, some comments.

File: src/browser/documentation/sidebar-guides/intro.tsx (hunk 3 above)
Comment (renetapopova) [reply src/browser/documentation/sidebar-guides/intro.tsx:47]: I see these "lead"s as subtitles, so no dots. 😄

File: src/browser/documentation/sidebar-guides/intro.tsx (hunk 4 above)
Comment (renetapopova) [reply src/browser/documentation/sidebar-guides/intro.tsx:96]: same here ... unless we decide the other way around ... but on some of the pages these leads are just a couple of words

File: src/browser/documentation/sidebar-guides/intro.tsx (hunk 5 above)
Comment (renetapopova) [reply src/browser/documentation/sidebar-guides/intro.tsx:103]: I can't see any difference between my suggestion and yours.

File: src/browser/documentation/sidebar-guides/intro.tsx (hunk 6 above)
Comment (renetapopova) [reply src/browser/documentation/sidebar-guides/intro.tsx:123]: no dot 😄

**Review by OskarDamkjaer — APPROVED** [review]

**Review by martin-neotech — APPROVED** [review]
Thanks.

**Pull request merged**